learning_paths/signals.py

```diff
@@ -0,0 +1,41 @@
+import logging
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from .compat import CourseEnrollment
+from .models import (
+    LearningPathCourseEnrollment,
+    LearningPathStep,
+    LearningPathEnrollment,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(post_save, sender=CourseEnrollment)
```

Comment:
Nit: how about using the [COURSE_ENROLLMENT_CREATED](https://github.com/openedx/openedx-events/blob/912ecc0e7cd58f3a8d520cc703e198f1299b6dcd/openedx_events/learning/signals.py#L56-L57) signal?

Reply:
I'm wondering if we should just do away with the CourseEnrollment foreign key and only use the COURSE_ENROLLMENT_CREATED and COURSE_UNENROLLMENT_COMPLETED signals to link the learning path enrollments to course enrollments. While the current approach works within lms, it breaks the modularity of the plugin and creates too much of a dependency on lms (CI tests can't pass without lms). What do you think?

Reply:
If we don't need the CourseEnrollement object, yeah, definitely.